tycho-its/src/test/java/org/eclipse/tycho/test/baseline/BaselineMojoTest.java

```diff
@@ -139,4 +206,42 @@ private Verifier getBaselineProject(String project) throws Exception {
 		verifier.addCliOption(project + "/pom.xml");
 		return verifier;
 	}
+
+	private static void setPomArtifactVersion(Path projectRoot, String version) throws IOException {
+		File pom = projectRoot.resolve("pom.xml").toFile();
+		Model model = new DefaultModelReader().read(pom, null);
+		model.setVersion(version);
+		new DefaultModelWriter().write(pom, null, model);
+	}
+
+	private static final Attributes.Name BUNDLE_VERSION = new Attributes.Name(Constants.BUNDLE_VERSION);
+
+	private static void setManifestBundleVersion(Path projectRoot, String version) throws IOException {
```

Comment:
Same here, enhancing this to other use case wil most probably require setting any manifest property.

Reply:
Done as well.